module-options.nix

```diff
@@ -86,32 +86,81 @@ in
           The treefmt package, wrapped with the config file.
         '';
         type = types.package;
+        default =
```

Comment:
```suggestion
        defaultText = lib.literalMD "wrapped `treefmt` command";
        default =
```

Reply:
or leave it in `config`

Reply:
I'm really curious about the purpose of `defaultText` here. Why add `defaultText` to `wrapper` but not `programs` even though both of them define a `default` value?

Reply:
The documentation can't display any packages. So this is to get the docs to build. I should've made that clear.
Feel free to add more defaultTexts, if you think it improves the docs.
If you don't think a default is relevant, you can move the definition to the `config` section, possibly with `lib.mkDefault` if you want to invite users to override it.
Alternatively, you can make options `internal = true` to remove them from the docs.